docker-for-ibm-cloud/administering-swarms.md

```diff
@@ -0,0 +1,266 @@
+---
+description: Administer swarm clusters with Docker EE for IBM Cloud
+keywords: ibm, ibm cloud, logging, iaas, tutorial
+title: Administer swarm clusters with Docker EE for IBM Cloud
+---
+
+Docker Enterprise Edition for IBM Cloud (Beta) comes with a variety of integrations that simplify the swarm administration process. Use the Docker EE for IBM Cloud CLI plug-in (`bx d4ic`) to quickly provision swarm mode clusters and resources. Manage your cluster with the `bx d4ic` plug-in and the Docker EE Universal Control Plane (UCP) browser-accessed interface.
+
+## Create swarms
```

Comment:
s/swarms/a swarm

Reply:
We usually use plurals in headings to avoid the extra article.